v1.22/eks/README.md

```diff
@@ -0,0 +1,43 @@
+# Conformance testing Amazon EKS
+
+## Setup EKS Cluster
+
+Setup EKS cluster as per the [EKS documentation](https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html).
+Use one of the following ways to create a cluster.
```

Comment:
please remove this line since we only use **eksctl** to create the cluster.

Reply:
removed line in updated commit.